master/docs/manual/cfg-workers-docker.rst

```diff
@@ -285,9 +285,10 @@ In addition to the arguments available for any :ref:`Latent-Workers`, :class:`Hy
 
 .. code-block:: python
 
-    import requests, urlparse
+    from future.moves.urllib.parse import urlparse
```

Comment:
I suggest not to use `future.moves` in the examples, rather use Python 2 or Python 3 style, or both and one leave commented:

```python
# On Python 2:
from urlparse import urlparse
# On Python 3:
# from urllib.parse import urlparse

import requests

r = requests.get("http://localhost:4040/api/tunnels/command_line").json()
masterFQDN = urlparse(r['public_url']).netloc
```